src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -274,14 +274,17 @@
     <value>Introduce field for all occurrences of '{0}'</value>
   </data>
   <data name="Introduce_local_for_all_occurrences_of_0" xml:space="preserve">
```

Comment:
loc strin gname needs to match texttual value.  so this needs to be `Introduce_local_for_all_reads_of_0`.  Also applies to any other loc changes you made.